examples/interactive/cli-platform/host.c

```diff
@@ -1,3 +1,3 @@
-extern int rust_main();
+extern unsigned char rust_main();
```

Comment:
nit: may be better to use `uint8_t` from `stdint` here, since technically `char` is compiler-specified and can be larger than a u8!